### ISSUE TITLE
Cloned files are not restored correctly

### ISSUE THREAD

**Issue by KlausiD** [notepad-plus-plus/notepad-plus-plus#10365]
### Steps to Reproduce the Issue

1. start npp++
2. open a file
3. clone the file in another view
-> the cloned File has the same buffer
-> all changes are displayed in both files at the same time
4. close npp++
5. start npp++
-> now the file is open in view 1 and view 2
-> but it is not the same buffer!
-> the files can now be changed in different ways. only after saving is it noticed in the other view that the file has been changed.

### Expected Behavior
After restoring cloned files
they should also use the same buffer again.

**Comment by mere-human** [comment]
This is probably a duplicate of https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10266
Though, I prefer the description here more as it is more laconic and precise.
